gradle-baseline-java/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
     compile 'net.ltgt.gradle:gradle-errorprone-plugin'
     compile 'org.apache.maven.shared:maven-dependency-analyzer'
     compile 'org.github.ngbinh.scalastyle:gradle-scalastyle-plugin_2.11'
+    implementation 'org.eclipse.jgit:org.eclipse.jgit'
```

Comment:
This was coming in from spotless before, but looks like they've hidden their implementation details now:

```

org.eclipse.jgit:org.eclipse.jgit:5.3.2.201906051522-r
\--- com.diffplug.spotless:spotless-lib-extra:1.24.1
     \--- com.diffplug.spotless:spotless-plugin-gradle:3.24.2
          \--- runtimeClasspath (requested com.diffplug.spotless:spotless-plugin-gradle)
```